internal/backends/winit/Cargo.toml

```diff
@@ -30,6 +30,10 @@ accessibility = ["dep:accesskit", "dep:accesskit_winit"]
 raw-window-handle-06 = ["winit/rwh_06"]
 default = []
 
+[lints.rust]
+# This might get set via RUSTFLAGS by users:
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(web_sys_unstable_api)'] }
```

Comment:
Since the winit backend has a build.rs, i have a slight preference for using the println incantation instead. (but that's a small preference)